frontend/packages/kubevirt-plugin/src/components/modals/vm-status-modal/vm-status-modal.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { Modal } from '@patternfly/react-core';
+import { HandlePromiseProps, withHandlePromise } from '@console/internal/components/utils';
+import { ModalComponentProps } from '@console/internal/components/factory';
+import { ModalFooter } from '../modal/modal-footer';
+import { PAUSED_VM_MODAL_MESSAGE } from '../../../constants/vm';
+import { VMIKind } from '../../../types';
+import { unpauseVM } from '../../../k8s/requests/vmi/actions';
+
+const modalTitle = 'Edit pause state';
+
+const VMStatusModalComponent = ({
+  vmi,
+  isOpen,
+  setOpen,
```

Comment:
it would be better to pass `close` which doesnt take any parameters. Same as I mentioned in Gilad's PR https://github.com/openshift/console/pull/3805#discussion_r368941069 - he is going to do that in a follow up so this should be done too

Reply:
Got it. I'll do this in a follow-up, as well.